interview-questions.md

```diff
@@ -7,52 +7,64 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props is how you pass information from the parent component to the child component. You make the information available by referencing the variable with props. 
 
-  Researched answer:
+  Researched answer: Props is a keyword in React, that stands for properties, and is being used for passing data from one component to another. The important part is that the data in props is being passed in a uni-directional flow (from parent to child). Props data is read-only, the data coming from the parent cannot be changed by the child component.
```

Comment:
Your opening looks very similar to  `“Props” is a special keyword in React, which stands for properties and is being used for passing data from one component to another. `